src/main/java/Nehsik.java

```diff
@@ -0,0 +1,93 @@
+import java.util.Scanner;
+public class Nehsik {
+    public static final int MAX_TASKS = 100;
```

Comment:
Good job!  The use of MAX_TASKS as a constant for the array size is good practice, making it easy to change the maximum number of tasks if needed.